operations/mimir-mixin/dashboards/tenants.libsonnet

```diff
@@ -530,5 +530,106 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Query Frontend Insights')
```

Comment:
I would split these in the other dimension, so have separate rows for:
- Queries (user)
- Queries (ruler)

And call the panels:
- Queries / sec
- Queue Length